isis/tests/CnetWinnowTests.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include <QTemporaryDir>
+
+#include "cnetwinnow.h"
+
+#include "PixelType.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+#include "FileName.h"
+#include "SerialNumber.h"
+
+#include "Fixtures.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetwinnow.xml").expanded();
```

Comment:
These should be defined in the app source/header instead of in the test files.

Reply:
These paths exist exclusively for tests, putting them elsewhere implies otherwise and not sure why we would want to do that. If we ever wanted a real interface to these apps it would include something that doesn't require users to know what this XML is or what it's for.